README.md

```diff
@@ -196,9 +196,9 @@ String or Array to specify address(es) for which sshd will bind. Corresponds to
 
 sshd_config_port
 ---------------------------
-String to specify listen port for sshd. Port option in sshd_config.
+Array to specify listen port[s] for sshd. Port option in sshd_config.
 
-- *Default*: '22'
+- *Default*: ['22']
```

Comment:
This would need a major version change as it changes the API in semver speak. To maintain backward compatibility sshd_config_port would need to support being a string or an array.